### ISSUE TITLE
Officially support more typical CMake build folder position

### ISSUE THREAD

**Issue by ssokolow** [autc04/Retro68#265]
`mkdir ../Retro68-build` needlessly clutters up my `~/src/read-only` folder by breaking the "one folder, one project" rule unless I add an extra layer of nesting to paths that can already get awkwardly long, so I decided to go for this more typical-for-CMake-projects layout instead:

```
cd /path/to/checkout/of/Retro68
mkdir build
cd build
../build-toolchain.bash
```

It seems to work fine for me. Could it be an officially supported option and noted as such in the README?

**Comment by autc04** [comment]
Yes, there is no reason any more why one shouldn't use a standard layout.
